atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitions.java

```diff
@@ -58,7 +58,8 @@ static CfDef getCfDef(String keyspace, TableReference tableRef, int gcGraceSecon
         Map<String, String> compressionOptions = getCompressionOptions(tableMetadata);
         cf.setCompression_options(compressionOptions);
 
-        if (tableMetadata.map(TableMetadata::isAppendHeavyAndReadLight).orElse(false)) {
+        boolean appendHeavyAndReadLight = tableMetadata.isPresent() && tableMetadata.get().isAppendHeavyAndReadLight();
```

Comment:
nit: `tableMetadata.map(TableMetadata::isAppendHeavyAndReadLight).orElse(false)` ??

Reply:
yeah, sure. Not sure why I changed this one back